### ISSUE TITLE
Menu component does not support scrollIntoView (CANVAS-1084)

### ISSUE THREAD

**Issue by jpante** [Workday/canvas-kit#808]
## 🐛 Bug Report

When using active descendant focus in a menu component that has more menu items than the height of that menu, or if text size is increased so the size of the menu increased, a keyboard user cannot scroll to see items positioned outside the menu container.

https://5d854c26ba934e0020f5e98a-lrapefiawj.chromatic.com/?path=/docs/labs-menu-react--default

This is because as the active descendant changes, it is programmatically exposed through the `aria-activedescendant ` property - the author is then responsible for ensuring the active descendant is visible to the user.sb

## Found In

3.x and 4.x

## Expected Behavior

The JavaScript `scrollIntoView` method is supported by all major browsers, and takes a Boolean parameter to indicate whether the element should be positioned at the top or bottom of the viewport - this is one way to address this.

**Accessibility SME** - @mathalete
